addons/calendar/wizard/calendar_alarm_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import datetime
+
+from odoo import fields, models, tools
+
+
+class TestCalendarAlarm(models.TransientModel):
+    _name = 'calendar.alarm.test'
+    _description = 'Test Reminder Wizard'
+
+    event_id = fields.Many2one('calendar.event', string='Event', required=True, ondelete='cascade')
```

Comment:
Missing a meaningful help attribute to help the user figure out what it is.
"We will use this sample event to notify you so you can check what it looks like.\nDon't worry, the event recipient will not receive anything."

Reply:
done